schema.graphql

```diff
@@ -42,6 +82,44 @@ type MasterContractApproval @entity {
   approved: Boolean!
 }
 
+type AssetTx @entity {
```

Comment:
I wonder in this case, since these entities are identical, that we introduce an enum for TransactionType, with options being something like "Asset", "Collateral", "Borrow". I wouldn't die on a hill over this, but I think it would allow for simpler queries with a single Transaction entity where you could query for all three, two, or a single type. What do you think about that?

Reply:
that makes sense :+1: